seed/seed.json

```diff
@@ -1219,6 +1219,27 @@
                 "channel": ["NIGHTLY"],
                 "platform": ["WINDOWS", "MAC", "LINUX"]
             }
+        },
+        {
+            "name": "BraveScreenFingerprintingBlockerRollout",
```

Comment:
Usually this ends with `Study`, i,e, `BraveScreenFingerprintingBlockerStudy`.

Reply:
I can use Study, although I thought maybe Rollout was a better descriptor because it's not merely a study in the strict sense that we are aiming to end at 100%.